easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb

```
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'axel'
+version = '2.17.9'
+
+github_account = "axel-download-accelerator"
+homepage = 'https://github.com/%(github_account)s/%(name)s'
+description = """Lightweight CLI download accelerator """
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/%(github_account)s/%(name)s/releases/download/v%(version)s']
+sources = [SOURCE_TAR_BZ2]
+checksums = ['d50dfbc59cb04fa70fb8d414579259c3eefe2a87aab52611309feeec9acb851a']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('pkg-config', '0.29.2'),
+    ('gettext', '0.20.1'),
+]
+
+dependencies = [('OpenSSL', '1.1.1d')]
+
+sanity_check_paths = {
+    'files': ["bin/axel"],
+    'dirs': [""],
+}
+
+sanity_check_commands = [True, ('%(name)s', '--version')]
```

I see that this format, `[True, ...]`, is also used in the `gzip` easyconfigs, but why?

---

@migueldiascosta `True` is equivalent to `%(name)s -h` here.

---

I would make that explicit:

```suggestion
sanity_check_commands = [
    "axel -h",
    "axel --version",
]